scss/common/_base.scss

```diff
@@ -387,6 +386,53 @@
         opacity: .3;
     }
 
+    $loading-size: 12px !default;
```

Comment:
Is this bound to the `$icon-size`? Something like `$icon-size - 2 * $loading-shadow-width`?